tests/utils/misc/misc.go

```diff
@@ -48,18 +48,13 @@ func WaitForExpectedState(fn getVMPowerStatus, vmName, expectedState string) boo
 	log.Printf("Timed out to poll status\n")
 	return false
 }
-// LogTestStart - Print a start log with given test name and
-// current time stamp
+
+// LogTestStart - Print a start log with given test group and test case name
 func LogTestStart(testGroup string, testName string) {
-	log.Printf("START:%s %s %s", testGroup, testName, curTime())
+	log.Printf("START: %s.%s", testGroup, testName)
```

Comment:
I don't follow this. As I've already explained in the PR description, go log itself will print out timestamp for each log message, for example:

2017/06/05 02:46:12 START: restart_test.TestPluginKill
2017/06/05 02:46:12 Attaching volume [restart_test_volume_1496630768] on VM[10.192.232.148]
2017/06/05 02:46:14 Confirming attached status for volume [restart_test_volume_1496630768]
2017/06/05 02:46:16 Killing vDVS plugin on VM [10.192.232.148]
2017/06/05 02:46:18 Sleep for 2 seconds

Why do we want to append a redundant timestamp?